core/js/src/main/scala/sigma/crypto/Platform.scala

```diff
@@ -246,7 +246,9 @@ object Platform {
     case c: Coll[_] => tpe match {
       case STuple(items) => c.tItem == sigma.AnyType && c.length == items.length
       case tpeColl: SCollection[_] => true
-      case _ => sys.error(s"Collection value $c has unexpected type $tpe")
+      case _ => sys.error(s"The value in the collection $c is not the correct type: $tpe. " +
+        "Please make sure the value is of the correct type. " +
+        "If the issue keeps happening, contact <a href=\"#\">Customer care</a>.")
```

Comment:
The meaning of the message has been changed, which makes it less clear (despite many more words).
Please see how the method is used to come up with a clarifying message.